rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -421,6 +424,21 @@ public void registerBtcTransaction(
         }
     }
 
+    private void processSvpFundTransactionHashSigned(BtcTransaction transaction) {
```

Comment:
Since this method is only called during an ongoing SVP period, can we safely call this `fundTx` instead of `transaction`?

Reply:
not really, because this method receives a btc transaction and then checks **if** it is the fund transaction.
Maybe i should change the method's name